src/helm/benchmark/run_specs.py

```diff
@@ -44,6 +45,7 @@
     TaskType,
     get_lextreme_task_type,
 )
+from .scenarios.cleva_scenario import CLEVAScenario
```

Comment:
Remove this import and move it to inside `get_cleva_spec`. Otherwise `ImportError` will be raised for users who don't install the optional dependencies.